src/main/kotlin/Connection.kt

```diff
@@ -95,6 +103,9 @@ class Connection(
             is ChannelParted -> {
                 users.remove(event.user.nickname)
                 addLine("${event.timestamp} -- ${event.user.nickname} Left${if (event.reason.isNotEmpty()) " ${event.reason}" else ""}")
+                if (event.user.nickname == client.serverState.localNickname) {
```

Comment:
```suggestion
                if (client.isLocalUser(event.user)) {
```

(Bit neater, and the IrcClient handles case sensitivity and stuff for you)